README.md

```diff
@@ -40,95 +42,85 @@ Many Go projects are built using Viper including:
 go get github.com/spf13/viper
 ```
 
-**Note:** Viper uses [Go Modules](https://go.dev/wiki/Modules) to manage dependencies.
+> **NOTE** Viper uses [Go Modules](https://go.dev/wiki/Modules) to manage dependencies.
```

Comment:
We could use the new highlight feature: https://github.com/orgs/community/discussions/16925